tests/kernel/threads/scheduling/schedule_api/testcase.yaml

```diff
@@ -1,4 +1,4 @@
 tests:
 -   test:
-        tags: kernel
+        tags: kernel threads sched
```

Comment:
nitpick: maybe 'scheduling' or 'scheduler'?

Reply:
IMHO, "sched" sounds good, because it's short. Make it longer, and people will wonder "Was it scheduler? Or scheduling?"